### ISSUE TITLE
Cloud. How to skip a native Cloud patch?

### ISSUE THREAD

**Issue by atwixfirster** [magento/devdocs#7605]
# Topic clarification/correction request

## Topic Link

[Apply patches](https://devdocs.magento.com/cloud/project/project-patch.html)

## Description

We have a topic about Cloud patches. However this topic does not contain an information how to skip a particular Cloud patch.

## Additional information

On our Cloud project we have removed Amazon payment modules via `replace` directive in a `composer.json` file:

```
"replace": {
        "amzn/amazon-pay-and-login-with-amazon-core-module": "*",
        "amzn/amazon-pay-module": "*",
        "amzn/amazon-pay-sdk-php": "*",
        "amzn/login-with-amazon-module": "*"
},
```

However, during deployment process Cloud returns a bunch of errors:

```
apply [--git-installation [GIT-INSTALLATION]]  
    W: 
    W: In Processor.php line 118:
    W:                                                                                
    W:   [12] The command "php ./vendor/bin/ece-patches apply" failed. In Applier.ph  
    W:   p line 140:                                                                  
    W:                                                                                
    W:                                                                                
    W:     The command "'git' 'apply' '--check' '--reverse' '/app/vendor/magento/mag  
    W:   en                                                                           
    W:     to-cloud-patches/patches/BUNDLE-2554__set_payment_info_bug_fix__3.4.1.pat  
    W:   ch                                                                           
    W:     '" failed.                                                                 
    W:                                                                                
    W:                                                                                
    W:                                                                                
    W:     Exit Code: 1(General error)    
```

**Problem** Cloud tries to patch a module which does not exist.

Do we have any docs how to disable a particular Cloud patch (in my case - `app/vendor/magento/mageto-cloud-patches/patches/BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch`) during deployment?

Thank you!

CC: @meker12

**Comment by atwixfirster** [comment]
Related issue: https://github.com/magento/magento-cloud-patches/issues/51

Thank you so much, @hostep

**Comment by meker12** [comment]
So you cannot skip patches required patches on Cloud projects. The issue you are experiencing is due to a bug that will be fixed in the next release of Cloud Patches (1.0.6).  Looks like bug affects installations with 2.2.6 and 2.3.5. See https://github.com/magento/magento-cloud-patches/pull/63.
